### PR TITLE
endpoints for speaking

### PR DESCRIPTION
**Pull request by Darina007**

### CODE REVIEW

File: src/main/java/com/team4/testingsystem/controllers/FileAnswerController.java [hunk 1]
```diff
@@ -24,27 +22,25 @@ public FileAnswerController(FileAnswerService fileAnswerService) {
         this.fileAnswerService = fileAnswerService;
     }
 
-    @ApiOperation(value = "Is used to get a file answer from the database by its id")
-    @GetMapping(path = "/{id}")
-    public FileAnswer get(@PathVariable("id") long id) {
-        return fileAnswerService.getById(id);
+    @GetMapping("/essay/{testId}")
```
Comment (AntonMaksimenkau) [inline src/main/java/com/team4/testingsystem/controllers/FileAnswerController.java]: the frontend shouldn't have a separate endpoint for getting essay. The data should be taken on our side and passed to DTO

File: src/main/java/com/team4/testingsystem/controllers/AnswerController.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/answer")
+public class AnswerController {
+
+    @GetMapping("/essay")
+    public String downloadEssay(@RequestParam Long testId) {
+        return "";
```
Comment (AntonMaksimenkau) [inline src/main/java/com/team4/testingsystem/controllers/AnswerController.java]: Is it a stub only?

File: src/main/java/com/team4/testingsystem/controllers/AnswerController.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/answer")
+public class AnswerController {
```
Comment (AntonMaksimenkau) [inline src/main/java/com/team4/testingsystem/controllers/AnswerController.java]: Why do we need a separate controller for it?

File: src/main/resources/db/changelog/changes/schema.xml [hunk 4]
```diff
@@ -607,4 +607,11 @@
         </addColumn>
     </changeSet>
 
+    <changeSet id="remove_id_from_file_answer" author="Darina">
+        <dropColumn tableName="file_answer" columnName="id"/>
+    </changeSet>
+
+    <changeSet id="remove_id_from_file_answer" author="Darina">
+        <dropColumn tableName="file_answer" columnName="id"/>
+    </changeSet>
```
Comment (AntonMaksimenkau) [inline src/main/resources/db/changelog/changes/schema.xml]: Why do we need 2 changesets for it?

**Review by AntonMaksimenkau — CHANGES_REQUESTED** [review]

**Review by AntonMaksimenkau — APPROVED** [review]

**Review by maxim12321 — APPROVED** [review]

**Pull request merged**